rpm/fedora/xournalpp.spec

```diff
@@ -0,0 +1,148 @@
+%global	build_repo https://github.com/xournalpp/xournalpp/
+%global	build_branch master
+%global	version_string 1.1.0
+%define	build_commit %(git ls-remote %{build_repo} | grep "refs/heads/%{build_branch}" | cut -c1-41)
+%define	build_shortcommit %(c=%{build_commit}; echo ${c:0:7})
+%global	build_timestamp %(date +"%Y%m%d")
+%global	rel_build %{build_timestamp}git%{build_shortcommit}%{?dist}
+
+Name:           xournalpp
+Version:        %{version_string}
+Release:        0.1.%{rel_build}
+Summary:        Handwriting note-taking software with PDF annotation support
+
+License:        GPLv2+
+URL:            https://github.com/%{name}/%{name}
+Source:         %{url}/archive/%{build_branch}.tar.gz
+
+BuildRequires:  cmake3 >= 3.10
+BuildRequires:  desktop-file-utils
+BuildRequires:  pkgconfig(cppunit) >= 1.12-0
+BuildRequires:  gcc-c++
+BuildRequires:  gettext
+BuildRequires:  git
+BuildRequires:  libappstream-glib
+BuildRequires:  pkgconfig(glib-2.0)
+BuildRequires:  pkgconfig(gtk+-3.0)
+BuildRequires:  pkgconfig(libxml-2.0)
+BuildRequires:  pkgconfig(libzip)
+BuildRequires:  pkgconfig(lua)
+BuildRequires:  pkgconfig(poppler-glib)
+BuildRequires:  pkgconfig(portaudiocpp) >= 12
+BuildRequires:  pkgconfig(sndfile)
+Requires:       hicolor-icon-theme
+Requires:       texlive-scheme-basic
```

Comment:
LaTeX is not strictly required for Xournal++ to function correctly (if it is, that's a bug that will need to be fixed).

Reply:
COPR repository is the equivalent of Ubuntu PPA allowing to build package without interfering to the main Fedora reposiotry. It is also useful for reviewing spec file for adherence to the Fedora Packaging Guideline.
I will update the spec setting LaTeX as an optional dependency.

Reply:
The PR is ready to merge. The rpm/fedora/xournalpp.spec is strictly reserved for the daily build from the master branch. LaTeX requirement is removed as the built was successful without it.